src/main/java/bio/terra/cli/command/shared/options/BqDatasetLifetime.java

```diff
@@ -0,0 +1,36 @@
+package bio.terra.cli.command.shared.options;
+
+import picocli.CommandLine;
+
+/**
+ * Command helper class that defines the default lifetime options for `terra resource` commands that
+ * handle Big Query Datasets.
+ *
+ * <p>This class is meant to be used as a @CommandLine.Mixin.
+ */
+public class BqDatasetLifetime {
+
+  @CommandLine.Option(
+      names = "--default-partition-lifetime",
+      description =
+          "The default lifetime, in seconds, for partitions in newly created partitioned tables. This flag has no minimum value. Specify 0 to remove the existing expiration time. Any partitions in newly created partitioned tables are deleted this many seconds after the partition's UTC date. This value is applied if you do not set a partition expiration on the table when it is created.")
```

Comment:
`UTC date` doesn't mean much to me. I see you're copying from the bq documentation, but either link it or explain in more detail.

Reply:
Please take a look at the updated description and see if I've wordsmithed it sufficiently to explain the relevance of UTC to the expiration.